### PR TITLE
OV Menu height should be dynamically calculated

### PR DESCRIPTION
**Pull request by Larisa-Staroverova**

### CODE REVIEW

File: feature-libs/product-configurator/rulebased/components/overview-menu/configurator-overview-menu.component.ts [hunk 1]
```diff
@@ -34,6 +34,7 @@ export class ConfiguratorOverviewMenuComponent implements AfterViewInit {
   protected readonly OV_MENU_ITEM = '-ovMenuItem';
   protected readonly OV_GROUP = '-ovGroup';
   protected readonly ACTIVE_CLASS = 'active';
+  protected readonly MENU_ITEM_HEIGHT = 39.5;
```
Comment (ChristophHi) [inline feature-libs/product-configurator/rulebased/components/overview-menu/configurator-overview-menu.component.ts:41]: Please document what this is about. I suppose this has a connection to styling attributes (we should mention this as well)

File: feature-libs/product-configurator/rulebased/components/service/configurator-storefront-utils.service.ts [hunk 2]
```diff
@@ -18,6 +18,7 @@ import { Configurator } from '../../core/model/configurator.model';
   providedIn: 'root',
 })
 export class ConfiguratorStorefrontUtilsService {
+  protected readonly CX_PREFIX = 'cx';
   protected readonly SEPARATOR = '--';
   protected readonly ADD_TO_CART_BUTTON_HEIGHT = 82;
```
Comment (ChristophHi) [inline feature-libs/product-configurator/rulebased/components/service/configurator-storefront-utils.service.ts:30]: Please document, how is it connected to styling?

File: feature-libs/product-configurator/rulebased/components/overview-menu/configurator-overview-menu.component.ts [hunk 3]
```diff
@@ -61,9 +62,19 @@ export class ConfiguratorOverviewMenuComponent implements AfterViewInit {
   }
 
   protected getHeight(): string {
```
Comment (ChristophHi) [inline feature-libs/product-configurator/rulebased/components/overview-menu/configurator-overview-menu.component.ts:78]: Please also document the meaning of that method

**Review by ChristophHi — CHANGES_REQUESTED** [review]

File: feature-libs/product-configurator/rulebased/components/overview-menu/configurator-overview-menu.component.ts (hunk 1 above)
Comment (Larisa-Staroverova) [reply feature-libs/product-configurator/rulebased/components/overview-menu/configurator-overview-menu.component.ts:41]: Done!

File: feature-libs/product-configurator/rulebased/components/overview-menu/configurator-overview-menu.component.ts (hunk 3 above)
Comment (Larisa-Staroverova) [reply feature-libs/product-configurator/rulebased/components/overview-menu/configurator-overview-menu.component.ts:78]: Done!

File: feature-libs/product-configurator/rulebased/components/service/configurator-storefront-utils.service.ts (hunk 2 above)
Comment (Larisa-Staroverova) [reply feature-libs/product-configurator/rulebased/components/service/configurator-storefront-utils.service.ts:30]: Done!

**Review by ChristophHi — APPROVED** [review]

**Pull request merged**